scripts/utils/gnosis_safe_server_interactions.js

```diff
@@ -13,6 +13,7 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
   const linkPrefix = {
     rinkeby: "rinkeby.",
     mainnet: "",
+    xdai: "xdai",
```

Comment:
```
$ curl  https://xdai.gnosis-safe.io/app/#/welcome
curl: (6) Could not resolve host: xdai.gnosis-safe.io
```
There is currently no frontend at this address, is this expected to change?

Reply:
It's good however that the [transaction service exists](https://safe-transaction.xdai.gnosis.io/api/v1/about/) .

Reply:
Yes, the frontend will be deployed ETA next week